davinci/src/main/kotlin/com/pingidentity/davinci/collector/DeviceAuthenticationCollector.kt

```diff
@@ -25,9 +25,10 @@ class DeviceAuthenticationCollector : FieldCollector<JsonObject>(), Submittable
 
     var value: Device? = null
 
-    override fun init(input: JsonObject) {
+    override fun init(input: JsonObject) : DeviceAuthenticationCollector{
```

Comment:
Super picky here... leave a space before the `{` ;)

Reply:
Updated.